enterprise/internal/insights/resolvers/aggregates_resolvers.go

```diff
@@ -23,6 +24,19 @@ const (
 	searchTimeLimitSeconds       = 2
 )
 
+// Possible reasons that grouping is disabled
+var invalidQueryMsg = "Grouping is disabled because the search query is not vaild."
+var fileUnsupportedFieldValueFmt = `Grouping by file is not available for searches with "%s:%s".`
+var authNotCommitDiffMsg = "Grouping by author is only available for diff and commit searches."
+var cgInvalidQueryMsg = "Grouping by capture group is only available for regex searches that contain a capturing group."
```

Comment:
```suggestion
var cgInvalidQueryMsg = "Grouping by capture group is only available for regexp searches that contain a capturing group."
```
I would say regexp here to keep consistent with `patterntype:regexp` (since `patterntype:regex` doesn't work on the UI)